### PR TITLE
New site based on Material Design for Bootstrap

### LINKED ISSUE #3: Enable signup for newsletter

### LINKED ISSUE #4: Headings and slogans for landing page

### LINKED ISSUE #5: Texts for landing page

### LINKED ISSUE #9: Implement final grid design

### PR DESCRIPTION
**Pull request by omarreiss**
@rkoster Ok, so I guess I had a hacknight... :computer: 

I think the site is perfectly responsive now. I've checked multiple devices in the chrome inspector.

You can checkout the branch and play around with the site a bit. See if everything works for you. See if you can break responsive design.

You can review the HTML I've written so far.

Things that still need to be done:
- [ ] Get the right link color in the footer
- [ ] Take a look at the Roboto font. I think the Roboto font we were using before was a little nicer. I've shipped the old font in this PR under `fonts/roboto` but I haven't tried including it yet. I am currently including Roboto draft, which sits in `fonts`.
- [ ] See if I can kick out all of the non-minified assets before merge.
- [ ] Move all inline CSS to a custom CSS file.
- [x] Finalize section about BOSH.
- [ ] Replace BOSH header with something proper.
- [x] add section about BOSH
- [x] apply latest design iteration
- [x] Icon sizes
- [x] Fix vertical alignment of logo on small screens.
- [x] Spacing in the modal (vertical spacing between email input and text)

Fixes #3 
Fixes #4 
Fixes #5 
Fixes #9

### CODE REVIEW

**Comment by rkoster** [comment]
On small screens (towards mobile) the vertical alignment of the dozer logo is a bit off.

**Comment by rkoster** [comment]
I have added a new design based on the current html in which I experimented with the whitespace/margins.

**Comment by omarreiss** [comment]
@rkoster Ok, worked four more hours on the flight. Added a section about BOSH. Please take a look at the latest version. I hacked something together to create a nice header for BOSH. I need you to come up with a proper design.

File: index.html [hunk 1]
```diff
@@ -1,29 +1,319 @@
...
+            <li><strong>Instant DevOps:</strong>BOSH has a holistic approach towards systems, bringing development and operations closer together. The demand for consistency requires operators and developers to work closer together to be able to understand properly how changes to an application are going to affect the bigger picture and vice versa.</li>
+          </ul>
+
+          <p>
+            The above should give a basic idea of what makes BOSH such a powerful tool. Also, it should be clear now that using BOSH will affect the way an operator goes about his operations. At Dozer, we believe that those using BOSH will always end up with better designed, more redundant and healthier systems. More info and documentation about BOSH can be found <a>here</a> (More links down below). The easiest way to get started is by using <a>BOSH Lite</a>. Once you're ready for production, check back here and we'll get you started in a minute!
```
Comment (rkoster) [inline index.html]: Dozer should be the easiest way to get started. Maybe better to not mention bosh-lite on our frontpage.

**Comment by rkoster** [comment]
It feels like a lot of text, I did a quick word count comparison between lattice.cf (276) consul.io (299) and us (668). Maybe we should move some of the bosh content on a deeplink. When we keep the first section (BOSH is an incredibly ... consistency amongst deployments) we would also endup around 300 words.

**Comment by omarreiss** [comment]
Agreed. Wanted to discuss that anyway. I might not need a header after all. I'll try something.

**Comment by rkoster** [comment]
On click animation on logo does not cover the full menu bar hight

**Comment by rkoster** [comment]
Use https://asciinema.org for short demo

**Comment by rkoster** [comment]
Add dozer enteprise section

**Comment by omarreiss** [comment]
@rkoster I've added the asciinema, seems to work fine. Could you please have a go at the enterprise section?

**Comment by rkoster** [comment]
Looks good to me :+1:

**Pull request merged**